_docker/drupal/drupal-filesystem/web/modules/custom/rhd_katacoda/rhd_katacoda.module

```diff
@@ -92,25 +93,3 @@ function rhd_katacoda_preprocess_node(array &$variables) {
     $variables['katacoda_course_info']['course_time'] = $course_time;
   }
 }
-
```

Comment:
This hook_preprocess_breadcrumb() will never execute because it was named incorrectly. To execute in this module, it should be `rhd_katacoda_preprocess_breadcrumb()`. This breadcrumb change is no longer necessary/desired, and we should clean up (remove) the code if we are not executing it.